pkg/download/automation/download.go

```diff
@@ -59,7 +60,9 @@ func Download(client *client.Client, projectName string, automationTypes ...conf
 			lg.Warn("No resource mapping for automation type %s found", at.Resource)
 			continue
 		}
-		response, err := client.List(context.TODO(), resource)
+		ctx, cancel := context.WithTimeout(context.Background(), time.Minute)
+		defer cancel()
```

Comment:
As we're doing this in a loop, we're only canceling the old timeouts once all configuration types are downloaded. Not a huge issue with 3(?) types, but in general I think it'd be better to cancel after the API call - e.g. by wrapping the API call in a method that builds context, defers cancel and returns whatever response and error were